src/IIIFPresentation/API.Tests/Integration/DeleteCollectionTests.cs

```diff
@@ -0,0 +1,236 @@
+        {
+            Id = "DeleteTester",
+            UsePath = true,
+            Label = new LanguageMap
+            {
+                {"en", new() {"update testing"}}
+            },
+            Thumbnail = "some/location",
+            Created = DateTime.UtcNow,
+            Modified = DateTime.UtcNow,
+            CreatedBy = "admin",
+            Tags = "some, tags",
+            IsStorageCollection = true,
+            IsPublic = false,
+            CustomerId = 1
+        };
+
+        await dbContext.Hierarchy.AddAsync(new Hierarchy
+        {
+            CollectionId = "DeleteTester",
+            Slug = "delete-test",
+            Parent = RootCollection.Id,
+            Type = ResourceType.StorageCollection,
+            CustomerId = 1,
+            Canonical = true
+        });
+
+        await dbContext.Collections.AddAsync(initialCollection);
+        await dbContext.SaveChangesAsync();
+
+        var deleteRequestMessage = HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Delete,
+            $"{Customer}/collections/{initialCollection.Id}");
+
+        // Act
+        var response = await httpClient.AsCustomer(1).SendAsync(deleteRequestMessage);
+
+        var fromDatabase = dbContext.Collections.FirstOrDefault(c => c.Id == initialCollection.Id);
+        var fromDatabaseHierarchy = dbContext.Hierarchy.FirstOrDefault(c => c.CollectionId == initialCollection.Id);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.NoContent);
+        fromDatabase.Should().BeNull();
+        fromDatabaseHierarchy.Should().BeNull();
+    }
+
+    [Fact]
+    public async Task DeleteCollection_FailsToDeleteCollection_WhenNotFound()
+    {
+        // Arrange
+        var deleteRequestMessage = HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Delete,
+            $"{Customer}/collections/doesNotExist");
+
+        // Act
+        var response = await httpClient.AsCustomer(1).SendAsync(deleteRequestMessage);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.NotFound);
+    }
+
+    [Fact]
+    public async Task DeleteCollection_FailsToDeleteCollection_WhenAttemptingToDeleteRoot()
+    {
+        // Arrange
+        var deleteRequestMessage = HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Delete,
+            $"{Customer}/collections/{RootCollection.Id}");
+
+        // Act
+        var response = await httpClient.AsCustomer(1).SendAsync(deleteRequestMessage);
+
+        var errorResponse = await response.ReadAsPresentationResponseAsync<Error>();
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+        errorResponse!.ErrorTypeUri.Should()
+            .Be("http://localhost/errors/DeleteCollectionType/CannotDeleteRootCollection");
+        errorResponse.Detail.Should().Be("Cannot delete a root collection");
+    }
+
+
+    [Fact]
+    public async Task DeleteCollection_FailsToDeleteCollection_WhenAttemptingToDeleteRootDirectly()
```

Comment:
This test looks identical to `DeleteCollection_FailsToDeleteCollection_WhenAttemptingToDeleteRoot()`